webapp/graphql/PostMutations.js

```diff
@@ -4,27 +4,43 @@ export default () => {
   return {
     CreatePost: gql`
       mutation (
+        $id: ID
         $title: String!
+        $slug: String
         $content: String!
         $categoryIds: [ID]
-        $image: ImageInput
         $groupId: ID
+        $postType: PostType
+        $eventInput: _EventInput
       ) {
         CreatePost(
+          id: $id
           title: $title
+          slug: $slug
           content: $content
           categoryIds: $categoryIds
-          image: $image
```

Comment:
Like i told on the other PRs this has been deleted somewhere in the process but is still needed